### ISSUE TITLE
TTv5 Q&A: Inaccessible Link

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#178]
Location: Trusted Tester Q&A - June 18, 2019 > Post Questions Here page
URL: https://dev.section508testing.net/mod/forum/discuss.php?d=20
Issue:  The DHS Accessibility HelpDesk link is not accessible via mouse or keyboard.

![image](https://user-images.githubusercontent.com/34749031/60102388-932b7f00-972b-11e9-9403-d084f21c9499.png)

**Comment by drewnielson** [comment]
@HD508Github, we probably don't need a separate issue for each of these instances. Can you edit the description in #171 to include this and any other instance? Then go ahead and close this issue. Could you also forward the issue to Tonya by email? She is the only one that can edit her posts.

**Comment by HD508Github** [comment]
Sure, no problem.  I initially started to report the issue this way, but wasn’t sure if it was known all the places the link resides.  Also, I did inform Tonya yesterday of the issue; but I’ll forward the issue to her also.


Kind Regards,
Halimatou(Halima)Diallo
Sr. Accessibility Technician
Supporting the Department of Homeland Security|MGMT/OCIO/OAST
Office: 202-815-5957
Email: Halimatou.Diallo@associates.hq.dhs.gov<mailto:Halimatou.Diallo@associates.hq.dhs.gov>
DHS Accessibility Help Desk: 202-447-0440 (Voice), 202-447-0582 (Fax)
Website: accessibility.dhs.gov<http://accessibility.dhs.gov/> | Email: accessibility@hq.dhs.gov<mailto:accessibility@hq.dhs.gov>
[cid:image001.png@01D3A09F.E6927B00]
This communication, along with any attachments, is covered by federal and state law governing electronic communications and may contain sensitive and legally privileged information. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, use or copying of this message is strictly prohibited.  If you have received this message in error, please reply immediately to the sender and delete this message.  Thank you.

**Comment by HD508Github** [comment]
Issue relates to Issue #171; so it was combined in Issue #171 to be remediated as a global fix.  Therefore, this issue is being closed.
